libs/constrained-generators/src/Constrained/Spec/Generics.hs

```diff
@@ -372,7 +372,7 @@ forAll' xs f = forAll xs $ \x -> match @fn @p x f
 
 -- | Like `constrained` but pattern matches on the bound `Term fn a`
 constrained' ::
-  forall fn a p.
```

Comment:
This is just to make sure the order of type arguments is the same in `constrained` and `constrained'`

Reply:
seems like a good idea